### ISSUE TITLE
Very stupid bots

### ISSUE THREAD

**Issue by t11tanYT** [Megabyte918/MultiOgar-Edited#613]
bots always split and "dance" 
Make bots split system smarter
if i have minions all bots always split into it

**Comment by Gigabyte918** [comment]
Duplicate!

**Comment by MastaCoder** [comment]
Cy@

**Comment by Luka967** [comment]
lel

**Comment by Megabyte918** [comment]
I'm planning on working on it once everything else is finished and there is some performance to spare for smarter AI. Also this is a duplicate of many other issues, so I am closing.
